src/runtime_src/tools/scripts/xrtdeps.sh

```diff
@@ -477,6 +477,10 @@ install()
         apt install -y "${UB_LIST[@]}"
     fi
 
+    if [ $FLAVOR == "ubuntu" ] && [ $MAJOR == 20 ]; then
```

Comment:
Would we be enabling this for all Ubuntu version from 20.xx going forward.   For example:
` if [ $FLAVOR == "ubuntu" ] && [ $MAJOR >= 20 ]; then`

Reply:
I wanted to tie this to 20.04 since it would ship a known version of clang-tidy for which we have tested this configuration. Once we know more we can relax this constraint.